src/Components/PaperEditor/TranscriptsContainer/LabelsList/LabelForm.js

```diff
@@ -48,8 +48,9 @@ const LabelForm = (props) => {
     setColor(randomColor());
   };
 
-  const handleColorPickerChangeComplete = () => {
-    setColor({ color: chroma(color.hex).name() });
+  const handleColorPickerChangeComplete = (e) => {
```

Comment:
What is `e`? Is that just the standard event variable passed into handlers?

Reply:
yes